.hygen/generate/document-resource/controllers/index.ejs.t

```diff
@@ -1,34 +1,34 @@
 ---
-to: "src/controllers/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.controller.ts"
+to: "src/controllers/<%= name %>.controller.ts"
```

Comment:
This make the file which will be generated write in wrong name 
Like if the model name is Car the file name will be
Car.controller.ts not car.controller.ts and that is wrong 
please solve this issue  @MahamdSirafi